tools/get_all_quad_ids.py

```diff
@@ -0,0 +1,256 @@
+import argparse
```

Comment:
Can this just be done as a modification to the original script? i.e. if --all-quads or something, your path is triggered?

Reply:
Will do.